servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -125,14 +126,17 @@ void setAuditRefObjectBits() {
         // the value is a comma separated list of supported objects:
```

Comment:
Please change the comment from  "by default we're only going to handle audit enabled roles" to "by default we're only going to handle audit enabled roles and groups"